models/odos/optimism/odos_optimism_trades.sql

```diff
@@ -1,103 +1,45 @@
 {{ config(
-    schema = 'odos_optimism'
-    ,alias = 'trades'
-    ,materialized = 'incremental'
-    ,file_format = 'delta'
-    ,incremental_strategy = 'merge'
-    ,unique_key = ['tx_hash', 'evt_index']
-    ,incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
-    )
+        alias = 'optimism_trades'
```

Comment:
We still need the schema here:
```suggestion
        schema = 'odos_optimism'
        , alias = 'trades'
```